pyface/util/_optional_dependencies.py

```diff
@@ -0,0 +1,40 @@
+
+# (C) Copyright 2005-2020 Enthought, Inc., Austin, TX
+# All rights reserved.
+#
+# This software is provided without warranty under the terms of the BSD
+# license included in LICENSE.txt and may be redistributed only under
+# the conditions described in the aforementioned license. The license
+# is also available online at http://www.enthought.com/licenses/BSD.txt
+#
+# Thanks for using Enthought open source!
+""" Utilities for handling optional import dependencies."""
+
+import contextlib
+import logging
+
+
+_PYFACE_LOGGER = logging.getLogger("pyface")
```

Comment:
This makes me a bit twitchy, having the logger being used hard-coded.  Perhaps better would be to pass in the logger instance to `optional_import` so it can use the logger of the module where `optional_import` is called from.

Reply:
It would also make testing easier